flocker/cli/test/test_cli_script.py

```diff
@@ -12,7 +13,7 @@ class FlockerCLITests(FlockerScriptTestsMixin, TestCase):
     command_name = u'flocker'
 
 
-class CLIOptionsTests(StandardOptionsTestsMixin, SynchronousTestCase):
```

Comment:
:grey_question:  No action required, but this seems non-obvious so I wanted to be sure I got it right. Feel free to add a comment or clean up if you agree that it's cleaner some other way.

Maybe I'm crazy, but shouldn't the TestCase be subclassed at the point of implementation of the test? I guess that would require a class factory so that the bare mixin wasn't run by trial :cry:. It seemed odd to me that for this and the above test I had to look at the implementation of the Mixin in order to determine if it was okay to use TestCase rather than AsyncTestCase. This also seems to indicate that these "Mixins" are not really what I would call Mixins... they seem to actually be parametrized testcase implementations. I guess that's known badness and we have ideas of how to do it in the future using testtools. Probably good enough for now.

One compromise that might not be the worst idea in the world:

in the definition of the mixin:
```
class StandardOptionsTestsMixin(object):
   BaseTestCase = testtools.TestCase
    ...
```

Then here:
```
class CLIOptionsTests(StandardOptionsTestsMixin, StandardOptionsTestsMixin.BaseTestCase):
  ...
```